CHANGELOG.md

```diff
@@ -3,6 +3,12 @@ aws-parallelcluster-node CHANGELOG
 
 This file is used to list changes made in each version of the aws-parallelcluster-node package.
 
+2.8.0
+-----
+
+**BUG FIXES**
```

Comment:
I would see it more as an enhancement for the ARM support

Reply:
Made the change